regression/contracts/invar_havoc_dynamic_multi-dim_array_partial_const_idx/test.desc

```diff
@@ -19,4 +19,9 @@ We _could_ do a more precise analysis in future where we only havoc
 `data[4][5][6]` but not `data[1][2][3]` since the latter clearly cannot be
 modified in the given loop.
 
-Blocked on issue #6021 -- alias analysis is imprecise and returns `unknown`.
+Blocked on #6326:
```

Comment:
Is this a PR or a GitHub issue? I'd just add the link here.

Reply:
GitHub numbers both issues and PRs on the same number line. They also have a uniform URL for both: `<repo>/issues/<id>`.

So #6326 is a PR, but the URL [`https://github.com/diffblue/cbmc/issues/6326`](https://github.com/diffblue/cbmc/issues/6326) would work just fine.